src/profile/Index.vue

```diff
@@ -1,89 +1,169 @@
 <template>
   <BasePage>
     <template v-slot:header>
+      <!-- Todo update other locales -->
       {{ $vuetify.lang.t('$vuetify.profile.index.header', $root.idpConfig.idpName) }}
     </template>
+    
+    <ProfileProgress :profile="{user: $user, alternates, mfa}"/>
 
-    <v-row>
-      <v-col cols="6">
-        <Attribute :name="$vuetify.lang.t('$vuetify.profile.index.username')" :value="$user.idp_username" sameline />
-        <Attribute :name="$vuetify.lang.t('$vuetify.profile.index.lastLogin')" :value="$user.last_login | format" sameline />
-        <Attribute :name="$vuetify.lang.t('$vuetify.profile.index.manager')" :value="$user.manager_email" sameline />
-      </v-col>
+    <h3 class="mt-6">Verily {{ $root.idpConfig.idpName }}</h3>
```

Comment:
Assuming "Verily" is the name of the application, that may also be worth moving to a config or translation file, so that it can be easily overridden for different instances of the IDP (in case they want a different application name).

Reply:
Great idea.